src/appchain/bridge/components/AppchainBridgeProvider.tsx

```diff
@@ -217,30 +219,25 @@ export const AppchainBridgeProvider = ({
     }));
   }, []);
 
-  const handleDeposit = useCallback(async () => {
+  const handleOpenExplorer = useCallback(() => {
     const blockExplorerUrl =
       from.id === base.id
         ? 'https://basescan.org/tx/'
         : 'https://sepolia.basescan.org/tx/';
```

Comment:
Nit: I know it's unchanged code but we can use this utility func here: 

```suggestion
    import { getChainExplorer } from '@/core/network/getChainExplorer';

    const blockExplorerUrl = `${getChainExplorer(from.id)}/tx/`;
```

Reply:
fixed, ty